src/app/main.cpp

```diff
@@ -579,6 +580,7 @@ int main( int argc, char *argv[] )
 
   bool myRestoreDefaultWindowState = false;
   bool myRestorePlugins = true;
+  bool myPython = true;
```

Comment:
Let's not let the "my" noise expand :laughing: 
```suggestion
  bool withPython = true;
```